### ISSUE TITLE
Atack of the pipelines, Part 4: Missing instructions for master branch

### ISSUE THREAD

**Issue by jtudelag** [rht-labs/enablement-docs#524]
**Is your feature request related to a problem? Please describe.**
When doing the [exec 2, part 4](https://rht-labs.github.io/enablement-docs/#/2-attack-of-the-pipelines/README?id=part-4-the-jenkinsfile) there are missing instructions for the master branch.

**Describe the solution you'd like**
Clearer  instructions on how to create thepipeline for master branch.

**Describe alternatives you've considered**
Maybe forcing a merge of the Jenkinsfile from develop into master.

**Additional context**
Students are a bit confused about these steps.

**Comment by tomchalmers** [comment]
Resolved via https://github.com/rht-labs/enablement-docs/pull/562

**Comment by tomchalmers** [comment]
Hi @jtudelag, 

I also noticed this whilst being a participant on the DO500 course this week in London led by @springdo and co. I have since made a change and raised the PR via: https://github.com/rht-labs/enablement-docs/pull/562

It doesn't seem to have linked this issue correctly despite the correct comments, nevertheless I believe the necessary changes have been made to the README file within my PR.

**Comment by xandradx** [comment]
I disagree a bit with that. Because having a pipeline without the Testing phase (Unit testing) will not be ready to be merge into master. The first working pipeline for master should be at the next exercise. "Revenge of the Automated Testing". Just my opinion
